mods/acrylic-effect-radius-changer.wh.cpp

```diff
@@ -0,0 +1,137 @@
+// ==WindhawkMod==
+// @id              acrylic-effect-radius-changer
+// @name            Acrylic Effect Radius Changer
+// @description     Allows the user to change the Acrylic effect blur radius
+// @version         1.0.0
+// @author          Dulappy
+// @github          https://github.com/Dulappy
+// @include         dwm.exe
+// ==/WindhawkMod==
```

Comment:
The mod seems to be x64-only, add `@architecture x86-64` to avoid compiling a 32-bit version.